app/contact/page.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
This is why your `/contact` doesn't work. Read the docs on routing:
https://nextjs.org/docs/app/building-your-application/routing